:arrow_up: (gcc@10): Bump avr-libc to v2.1.0

ladislas opened the pull request.

ladislas committed:
:arrow_up: (gcc@10): Bump avr-libc to v2.1.0

ladislas commented:
@fauxpark any idea what could go wrong?

fauxpark commented on Formula/avr-gcc@10.rb:
```diff
@@ -44,9 +46,8 @@ class AvrGccAT10 < Formula
-    mirror "https://download-mirror.savannah.gnu.org/releases/avr-libc/avr-libc-2.0.0.tar.bz2"
-    sha256 "b2dd7fd2eefd8d8646ef6a325f6f0665537e2f604ed02828ced748d49dc85b97"
+    url "https://github.com/avrdudes/avr-libc/archive/refs/tags/avr-libc-2_1_0-release.tar.gz"
+    sha256 "97a5ffb25ad7ca0a241e8a73167fed8cbef6a2472822ccb697cb0df70f51e836"
```
```suggestion
    url "http://download.savannah.gnu.org/releases/avr-libc/avr-libc-2.1.0.tar.bz2"
    mirror "https://download-mirror.savannah.gnu.org/releases/avr-libc/avr-libc-2.1.0.tar.bz2"
    sha256 "0b84cee5c08b5d5cba67c36125e5aaa85251bc9accfba5773bfa87bc34b654e8"
```

From what I can tell, the archives from GitHub are just a straight rip of the repo - `./bootstrap` hasn't been run to generate the `configure` script etc.

The releases from Savannah appear to have this step already done.

ladislas replied:
Thanks so much! Trying that now :)

ladislas committed:
Use avr-libc from savannah

the archives from GitHub are just a straight rip of the repo - ./bootstrap hasn't been run to generate the configure script etc.

The releases from Savannah appear to have this step already done.

Co-authored-by: Ryan <fauxpark@gmail.com>

ladislas committed:
Use https
